.circleci/config.yml

```diff
@@ -47,7 +47,7 @@ workflows:
   weekly:
     triggers:
       - schedule:
-          cron: "0 0 18,20,24 * *" # FIXME "0 0 * * 3"
+          cron: "0 3 * * 4"
```

Comment:
The syntax is quite strange imo, should we add an inline comment (same below)?

Reply:
I'll add a link to https://crontab.guru/